test/e2e/broken-links.spec.ts

```diff
@@ -17,6 +17,10 @@ const falsePositives = [
   'https://www.nuget.org/packages/MaxMind.GeoIP2/',
   'https://www.nuget.org/packages/MaxMind.MinFraud',
   'https://www.rubydoc.info/gems/minfraud/',
+
+  // The pages in previews have a <link ..> in the header
+  // that points to the page on production. So, the page in the PR fails.
+  'https://dev.maxmind.com/geoip/license-key-validation-api',
```

Comment:
Looks like it is coming from the canonical url https://github.com/maxmind/dev-site/blob/267e5dfd9abb8628effceb525a4480d6baad876f/src/components/Layout/Seo.tsx#L44-L50

I can create a story for the test runner to update `site.siteMetadata.siteUrl` during the tests.

Reply:
I created https://www.pivotaltracker.com/story/show/187152961